pkgs/native_assets_builder/lib/src/model/build_dry_run_result.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:native_assets_cli/native_assets_cli_internal.dart';
+
+import '../../native_assets_builder.dart';
+
+/// Similar to a [BuildResult], but for the case of a dry run, where not assets
+/// are actually produced.
```

Comment:
I don't understand this sentence: "where not assets are actually produced".

Reply:
Updated the comment.